results/19-SelectedLatentVar-metaviperCor.Rmd

```diff
@@ -0,0 +1,116 @@
+---
+title: "19-Selected LV-Metaviper Cors"
```

Comment:
Is this file supposed to be here? Or is this just a differently-named dupe of the other 19 rmd?

Reply:
it sort of replaces the other - i ended up with branching issues.